src/sidebars/scrollbar-directive.js

```diff
@@ -7,6 +7,12 @@
                 restrict: 'C',
                 link: (scope, element) => {
                     var scrollPadding = 'scroll-padding';
+                    scope.$on('disableScrolling', function(event) {
+                        element.mCustomScrollbar('disable');
+                    });
+                    scope.$on('enableScrolling', function(event) {
+                        element.mCustomScrollbar('update');
+                    });
```

Comment:
Doesn't look like you need either of the `event` variables here. Would prefer to see `() => ` too :)